setup.cfg

```
@@ -1,11 +1,11 @@
 # setuptools config
 # see http://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files
 [metadata]
-name = open_forms_ext_token_exchange
+name = token_exchange
```

```suggestion
name = openforms-ext-token-exchange
```
I think if we decide to publish this on PyPI at some point, that would be the best pattern for package name

Otherwise this does not affect this project/repository at all